### PR TITLE
Corrections for HCAL iterative phi symmetry calibration AlCaReco .py files

### PR DESCRIPTION
**Pull request by nilina**
3 python files were added for AlCaReco stream:
   Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIterativePhiSym_Output_cff.py
   Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIterativePhiSym_cff.py
   Calibration/HcalAlCaRecoProducers/python/alcaiterphisym_cfi.py

and 1python file for testing:
     Calibration/HcalAlCaRecoProducers/test/AlCaIterativePhiSym_cfg.py

No any corrections and changes for producers!

### CODE REVIEW

**Comment by mmusich** [comment]
@cmsbuild please test

**Comment by diguida** [comment]
please test
The Frontier timeouts are now over

File: Calibration/HcalAlCaRecoProducers/test/AlCaIterativePhiSym_cfg.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("MYITERATIVEPHISYM")
+
+process.load("Configuration.Geometry.GeometryIdeal_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```
Comment (mmusich) [inline Calibration/HcalAlCaRecoProducers/test/AlCaIterativePhiSym_cfg.py]: please use the `condDBv2` version

File: Calibration/HcalAlCaRecoProducers/test/AlCaIterativePhiSym_cfg.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag=autoCond['startup']
```
Comment (mmusich) [inline Calibration/HcalAlCaRecoProducers/test/AlCaIterativePhiSym_cfg.py]: please use the `condDBv2` version
The correct key for `autoCond_condDBv2` dictionary is `run2_mc`.

**Comment by mmusich** [comment]
@nilina 
my understanding is that you want to add a new `ALCARECOHcalCalIterativePhiSym`, ALCARECO stream.
If you want this to be run in production you need to update:
`Configuration/EventContent/python/AlCaRecoOutput_cff.py`
`Configuration/EventContent/python/EventContent_cff.py`
and I guess `Configuration/StandardSequences/python/AlCaRecoStream_Specials_cff.py`

**Comment by diguida** [comment]
@nilina please rebase

**Comment by mmusich** [comment]
@nilina this does not merge anymore, please rebase

**Comment by nilina** [comment]
Could you please tell me how I can rebase it?

**Comment by nclopezo** [comment]
Hi @nilina,
You can see instructions here: https://cms-sw.github.io/tutorial-forward-port.html

**Comment by nilina** [comment]
I followed the instructions from the tutorial. Could you please whether I do it correctly.

**Comment by slava77** [comment]
-1

AlCa step is broken 

```
StandardSequences/AlCaRecoStream_Specials_cff.py", line 48, in <module>
    from Calibration.EcalAlCaRecoProducers.ALCARECOEcalCalPhiSym_cff import *
ImportError: No module named ALCARECOEcalCalPhiSym_cff

```

**Comment by mmusich** [comment]
-1 
@nilina it seem like you rebased before this commit https://github.com/bmarzocc/cmssw/commit/9a8d1652415da040d16ccf792581289d9e0aaef8 in PR #8678 from @bmarzocc removing the ECAL ALCARECOEcalCalPhiSym. Not completely clear to me how this happened, but please start again from a clean branch

**Comment by nilina** [comment]
I tried to fix problems. Could you please check it. Thanks.

**Comment by mmusich** [comment]
-1
@nilina this has still issues in list of commits.
May I suggest to close it and start from scratch starting from the lastest IB?
Thanks

**Comment by mmusich** [comment]
superseded by #9075 
@diguida thanks

**Pull request closed without merging**